demo/chbench/kafkasnap/snapshot_topics

```diff
@@ -0,0 +1,43 @@
+#!/usr/bin/env bash
+#
+# Copyright Materialize, Inc. All rights reserved.
+#
+# Use of this software is governed by the Business Source License
+# included in the LICENSE file at the root of this repository.
+#
+# As of the Change Date specified in that file, in accordance with
+# the Business Source License, use of this software will be governed
+# by the Apache License, Version 2.0.
+
+# Script to list and dump all topics (and their schemas) from a Kafka cluster
+
+if ! topics=$(kafkacat -L -b kafka:9092 | grep debezium | cut -d '"' -f 2); then
+    echo "Failed to list topics!"
+    exit 1
+fi
+
+SNAPSHOT_DIR="/snapshot"
+
+jq_filter=".schema"
+for topic in $topics; do
+    schemafile="${SNAPSHOT_DIR}/${topic}.schema"
+    if ! subject=$(curl -s schema-registry:8081/subjects/${topic}-value/versions/latest); then
+        echo "Failed to query subject ${topic}"
+        exit 1
+    fi
+
+    if ! jq -r "${jq_filter}" <<< "${subject}" > "${schemafile}"; then
```

Comment:
We can remove the `-r` flag here, so that schema stays as an escaped json string, with surrounding quotes. We can then remove the `schema_escaped` variable above and remove the extra quotes around it. Everything should work the same after doing that.